.github/workflows/semantic-pull-request.yml

```diff
@@ -47,3 +48,4 @@ jobs:
             utils
             wallet
             zmq
+            stats
```

Comment:
should update https://github.com/dashpay/dash/blob/master/CONTRIBUTING.md#creating-the-pull-request